src/test/java/nl/hsac/fitnesse/fixture/util/BsnUtilTest.java

```diff
@@ -2,24 +2,36 @@
 
 import org.junit.Test;
 
+import static junit.framework.TestCase.assertFalse;
```

Comment:
`junit.framework` is deprecated. Maybe you can configure your IDE to only use the `org.junit...` versions?

Reply:
Was a mistake to import this one...Thanks for tip.